src/storage/storage_backend.rs

```diff
@@ -160,6 +164,32 @@ pub trait StorageBackend<U: Sync + Send + Debug>: Send + Sync + Debug {
     /// [`Metadata`]: ./trait.Metadata.html
     async fn metadata<P: AsRef<Path> + Send + Debug>(&self, user: &Option<U>, path: P) -> Result<Self::Metadata>;
 
+    /// Returns the `Md5` for the given file.
```

Comment:
This `Md5` should be in CAPS here i.e. `MD5`

Reply:
yes